building_sim_plugins/building_sim_ign/src/door.cpp

```diff
@@ -140,26 +31,27 @@ class IGNITION_GAZEBO_VISIBLE DoorPlugin
 private:
   rclcpp::Node::SharedPtr _ros_node;
   Model _model;
+  std::unordered_map<std::string, std::shared_ptr<Entity>> _joints;
```

Comment:
At the end of the day Entity is just a uint64_t so I think it's fine to copy it around instead of making it a pointer